config/decode.go

```diff
@@ -41,26 +40,6 @@ type DecoderOptions struct {
 // envDecoder implements Decoder.
 type envDecoder struct{}
 
-// Decode populates target from the environment.
-// The target argument must be a pointer to a struct type value.
-func (e *envDecoder) Decode(opts *DecoderOptions, target interface{}) error {
```

Comment:
Not sure but can we also remove from this file:

* `envDecoder` struct 
* `Prefix` field from `DecoderOptions`? I think prefix is only used for env decoding...? Then we could simply pass `hcl.Body` to decoding function.

Reply:
Thanks @pondzix ! Done.